### DISCUSSION TITLE
Dead time change?

### DISCUSSION THREAD

**Discussion by majekw** [Ralim/IronOS#1210] [Q&A]
Hello,

where and how can I increase dead time in code? I mean time between switching off power and start of ADC conversion?
My hardware for tests is Pinecil if it helps :-)

**Comment by Ralim** [comment]
Hia, you want `holdoffTicks` here : https://github.com/Ralim/IronOS/blob/e18c877a96728bdc5810c9b4d9da58ea241a0ef8/source/Core/BSP/Pine64/BSP.cpp#L15

**Reply by majekw** [reply]
Thanks! That was fast :-)
Are some constrains or other things that could blow up if I change this?

**Reply by Ralim** [reply]
If you go to small it will not settle so temp will over ready, if you increase it you reduce max power
